docs/advanced/smart_memory_estimation.md

```diff
@@ -22,6 +22,8 @@ Percona XtraBackup performs the backup procedure in two steps:
 
 Starting with [Percona XtraBackup 8.0.30-23](../release-notes/8.0/8.0.30-23.0.md), while copying redo log entries during the backup phase, Percona XtraBackup computes the memory required for `prepare` phase. Percona XtraBackup also takes into consideration the number of InnoDB pages to be fetched from the disk. Then Percona XtraBackup checks the server's available free memory and uses that memory up to the limit specified in the `--use-free-memory-pct` option to run `--prepare`. The default value of `--use-free-memory-pct` option is 0. This means that the option is disabled by default. If, for example, you set the option to 50, 50% of the free memory is used to `prepare` a backup. The maximum allowed value is 100.
 
+Starting with Percona XtraBackup 8.0.31-24, use the `--estimate-memory` option to enable or disable the memory estimation. The default value is `OFF`. To enable the `Smart memory estimation` feature, set the  `--estimate-memory` option to `ON`.
```

Comment:
Starting with Percona XtraBackup 8.0.31-24, the `Smart memory estimation` feature can be enabled or disabled with the `--estimate-memory` option. The default value is `OFF`.  Enable the feature with `--estimate-memory=ON`.

Reply:
Updated